recipe/meta.yaml

```diff
@@ -1,38 +1,61 @@
-{% set version = "3.0.0b2.post2" %}
+{% set version = "3.0.0b2" %}
+{% set version_str = "v3.0.0-beta-2" %}
 
 package:
     name: protobuf
     version: {{ version }}
 
 source:
-    fn: protobuf-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/p/protobuf/protobuf-{{ version }}.tar.gz
-    md5: 42aa7ab79ebcb94917e938ff482ef7a6
+    fn: protobuf_{{ version }}.tar.gz
+    url: https://github.com/google/protobuf/archive/{{ version_str }}/protobuf-{{ version_str }}.tar.gz
+    md5: e7f2602baffcbc27fb607de659cfbab6
 
 build:
-    number: 2
+    number: 3
+    features:
+        - vc9                # [win and py27]
+        - vc10               # [win and py34]
+        - vc14               # [win and py35]
 
 requirements:
     build:
+        - git
+        - cmake              # [win]
+        - autoconf           # [unix]
+        - automake           # [unix]
+        - libtool            # [unix]
+        - pkg-config         # [unix]
+        - zlib
         - python
-        - setuptools
+        - six
+        - ordereddict        # [py26]
+        - unittest2          # [py26]
+
     run:
+        - zlib
         - python
-        - setuptools
         - six
+        - ordereddict        # [py26]
+        - unittest2          # [py26]
 
 test:
+    commands:
+        - protoc --help
+        - test -f ${PREFIX}/lib/libprotobuf.a                           # [unix]
+        - test -f ${PREFIX}/lib/libprotobuf.dylib                       # [osx]
+        - test -f ${PREFIX}/lib/libprotobuf.so                          # [linux]
+        - if not exist %PREFIX%\\Library\\lib\\libprotobuf.lib exit 1   # [win]
```

Comment:
Added a test for the main Windows static library. Again the default is static libraries. We could switch Windows to make dynamic libraries, but it is [one or the other]( https://github.com/google/protobuf/blob/master/cmake/CMakeLists.txt#L97-L100 ).